src/app/users/users.component.html

```diff
@@ -75,19 +75,19 @@
       </ng-container>
       <ng-container matColumnDef="name">
         <mat-header-cell *matHeaderCellDef mat-sort-header i18n>User Name</mat-header-cell>
-        <mat-cell *matCellDef="let element">{{element.doc.name}}</mat-cell>
+        <mat-cell *matCellDef="let element" (click)="gotoProfileView(element.doc.name)">{{element.doc.name}}</mat-cell>

```

Comment:
Please change the click event to apply to the `mat-row` element so we only include it once.